docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/install-with-rmp-deb.md

```diff
@@ -0,0 +1,78 @@
+# **Nebula Graph** Installation with rpm/deb Package
+
+## Overview
+
+This guide will walk you through the process of installing **Nebula Graph** with `rpm/deb` packages.
+
+## Prerequisites
+
+Before getting started, ensure that you meet the following requirements:
+
+* Hard disk: 50 GB
+* Memory: 8 GB
+
+## Installing **Nebula Graph**
+
+To install **Nebula Graph** with a `rpm/deb` package, you must complete the following steps:
+
+1. Log in to GitHub and click [rpm/deb](https://github.com/vesoft-inc/nebula/actions) to locate the `rpm/deb` package.
+
+2. Under the **Actions** tab, click **package** on the left. All packages available are displayed.
+
+3. Click the latest package on the top of the package list.
+
+![action-page](https://user-images.githubusercontent.com/40747875/71390992-59d1be80-263d-11ea-9d61-1d7fbeb1d8c5.png)
+
+4. Click the **Artifacts** list on the upper right corner to select a package to download.
+
+![select-a-package](https://user-images.githubusercontent.com/40747875/71389414-415ea580-2637-11ea-8930-eaef1e8a5d17.png)
+
+5. Install **Nebula Graph**.
+
+* For a `rmp` file, install **Nebula Graph** with the following command:
```

Comment:
rpm

Reply:
After the package is extracted, there is a file in rpm format. So, i would prefer to use rpm file.